nixos/modules/services/security/vaultwarden/default.nix

```diff
@@ -194,8 +194,32 @@ in {
         ProtectHome = "true";
         ProtectSystem = "strict";
         AmbientCapabilities = "CAP_NET_BIND_SERVICE";
-        StateDirectory = "bitwarden_rs";
+        StateDirectory = "vaultwarden";
```

Comment:
This change is going to break preexisting vaultwarden deployments.
Please make this value dependent on the set `system.stateVersion`.

Reply:
https://github.com/NixOS/nixpkgs/pull/292485/commits/52b8a9de611193327fb0369b200b538ac63d609f
feedback appreciated